staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,10 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
Well, both yes and no. I believe those priorities is copied from what `kubeApiServer` already has. Problem is we do not register those separately with aggregator. We get spec from `kubeApiServer` that has all of those. We can change implementation a little to take those into account but I think it is out of this PR (This PR does not actually change any priority from what it was before).